.config/git-merge-flow-config.jsonc

```diff
@@ -0,0 +1,38 @@
+// IMPORTANT: This file is read by the merge flow from main branch only. 
+{
+    "merge-flow-configurations": {
+        // MSBuild servicing chain from oldest supported through currently-supported to main
+        // Automate opening PRs to merge msbuild's vs16.11 (VS until 4/2029) into vs17.0 (SDK 6.0.1xx)
+        "vs16.11": {
+            "MergeToBranch": "vs17.0"
+        },
+        // Automate opening PRs to merge msbuild's vs17.0 (SDK 6.0.1xx) into vs17.3 (SDK 6.0.4xx)
+        "vs17.0": {
+            "MergeToBranch": "vs17.3"
+        },
+        // Automate opening PRs to merge msbuild's vs17.3 (SDK 6.0.4xx) into vs17.4 (SDK 7.0.1xx until 5/2024, VS until 7/2024)
+        "vs17.3": {
+            "MergeToBranch": "vs17.4"
+        },
+        // Automate opening PRs to merge msbuild's vs17.4 into vs17.6 (VS until 1/2025)
+        "vs17.4": {
+            "MergeToBranch": "vs17.6"
+        },
+        // Automate opening PRs to merge msbuild's vs17.6 into vs17.8 (VS until 7/2025)
+        "vs17.6": {
+            "MergeToBranch": "vs17.8"
+        },
+        // Automate opening PRs to merge msbuild's vs17.8 (SDK 8.0.1xx) into vs17.9 (SDK 8.0.2xx)
+        "vs17.8": {
+            "MergeToBranch": "vs17.9"
+        },
+        // Automate opening PRs to merge msbuild's vs17.9 (SDK 8.0.2xx) into vs17.10 (SDK 8.0.3xx)
+        "vs17.9": {
+            "MergeToBranch": "vs17.10"
+        },
+        // MSBuild latest release to main
+        "vs17.10": {
```

Comment:
It might be out of date due to undergoing release,  @AR-May please validate

Reply:
True, the step of updating the merge flow: https://github.com/dotnet/msbuild/issues/10248 is not completed yet :) 
@AR-May please let me know if you want me to update it here, or it could be done as part of the release steps.

Reply:
yes, we need merge vs17.10 -> vs17.11 -> main

Reply:
Let's update it here. Does this new flow configuration already work?

Reply:
This is updated. 
Configuration will not trigger any flow however the worflow file into the vs17.11 will be created soon. Thank you :)

Reply:
@AR-May , could you please add it to the release checklist template: https://github.com/dotnet/msbuild/blob/main/documentation/release-checklist.md ?